node-tests/babel-plugin-strip-wrappers-test.js

```diff
@@ -0,0 +1,175 @@
+const { expect } = require('chai');
```

Comment:
Why do we need to use mocha/chai for the node tests? We already use qunit here, and I'd rather we don't use both in the same project. Can we just use qunit?

Reply:
Yes!